task_templates/pipelines/python3_calibrated_anomaly_detection/custom.py

```diff
@@ -1,40 +1,111 @@
+"""
+    In this example we show how to create an unsupervised anomaly detection model
+    with calibrated predictions
+
+    Note: in our score hook we use the same column name as regression, i.e. columns=['Predictions']
```

Comment:
instead of here, shall we describe it where we describe what the score hook returns?